app/models/sc_manifest.rb

```diff
@@ -79,32 +78,32 @@ def convert_with_collection(user, collection)
 
     self.work = work
     self.save!
-    
+
     work
   end
 
   def cleanup_label(label)
     label = flatten_element(label)
     new_label = label.truncate(255, separator: ' ', omission: '')
-    new_label.gsub!("&quot;", "'")
-    new_label.gsub!("&amp;", "&")
-    new_label.gsub!("&apos;", "'")
+    new_label.dup.gsub!("&quot;", "'")
```

Comment:
I don't understand why we're duplicating a string and then modifying the duplication.  Shouldn't we be modifying the label itself?  I feel like this needs testing if the test suite doesn't already test it.  (Can we use Coveralls to find out whether this is being tested?)

Reply:
@saracarl I can't remember exactly why I did this, I just reverted it and the tests are still passing locally.